### Hello!
***

This is my first little experiment with python and jupyter notebooks. Let's see what we can do!

In [1]:
print("Hello World!")

Hello World!


Well that was a little too easy...

In [2]:
import arcpy

ModuleNotFoundError: No module named 'arcpy'

Well **fuck**, that is going to be a *problem*! I need arcpy... so now what?

In [6]:
import os, sys, datetime, pandas

#### Cool
I *guess* I am happy that some of the packages can be imported.  Just not the one I need the most.

In [3]:
import sys
print(sys.version)

3.6.4 |Anaconda, Inc.| (default, Jan 16 2018, 10:22:32) [MSC v.1900 64 bit (AMD64)]


#### AH-HA!!!! 
The version is 3.6 for Anaconda. That is why ArcPy is not included on this notebook